README.md

```diff
@@ -1020,70 +789,100 @@ Were a PITM attack successfully performed on a UCAN delegation, the proof chain
+[Meiklejohn]: https://christophermeiklejohn.com/
 [Mikael Rogers]: https://github.com/mikeal/
+[Multibase]: https://github.com/multiformats/multibase
+[Multicodec]: https://github.com/multiformats/multicodec
+[Multics]: https://en.wikipedia.org/wiki/Multics
+[Multihash]: https://www.multiformats.io/multihash/
 [OCAP]: http://erights.org/elib/capability/index.html
 [OCapN]: https://github.com/ocapn/ocapn
-[POLA]: https://en.wikipedia.org/wiki/Principle_of_least_privilege
+[P-256]: https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-4.pdf#page=111
+[PACELC]: https://en.wikipedia.org/wiki/PACELC_theorem
 [Philipp Krüger]: https://github.com/matheus23
+[PoLA]: https://en.wikipedia.org/wiki/Principle_of_least_privilege
 [Protocol Labs]: https://protocol.ai/
+[PvH]: https://www.pvh.ca
+[RBAC]: https://en.wikipedia.org/wiki/Role-based_access_control
 [RFC 2119]: https://datatracker.ietf.org/doc/html/rfc2119
 [RFC 3339]: https://www.rfc-editor.org/rfc/rfc3339
 [RFC 8037]: https://datatracker.ietf.org/doc/html/rfc8037
-[SHA2-256]: https://en.wikipedia.org/wiki/SHA-2
+[RSM]: https://en.wikipedia.org/wiki/State_machine_replication
+[Robust Composition]: http://www.erights.org/talks/thesis/markm-thesis.pdf
+[SHA-256]: https://en.wikipedia.org/wiki/SHA-2
 [SPKI/SDSI]: https://datatracker.ietf.org/wg/spki/about/
 [SPKI]: https://theworld.com/~cme/html/spki.html
 [Seitan token exchange]: https://book.keybase.io/docs/teams/seitan
 [Steven Vandevelde]: https://github.com/icidasset
-[Token Uniqueness]: #622-token-uniqueness
+[Storacha]: https://storacha.network/
+[The Structure of Authority]: http://erights.org/talks/no-sep/secnotsep.pdf
+[The computer revolution hasn't happened yet]: https://www.youtube.com/watch?v=oKg1hTOQXoY
+[UCAN Promise]: https://github.com/ucan-wg/promise
 [URI]: https://www.rfc-editor.org/rfc/rfc3986
+[Varsig]: https://github.com/ChainAgnostic/varsig
 [Verifiable credentials]: https://www.w3.org/2017/vc/WG/
 [W3C]: https://www.w3.org/
+[WebCrypto API]: https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API
+[Witchcraft Software]: https://github.com/expede
 [ZCAP-LD]: https://w3c-ccg.github.io/zcap-spec/
-[`did:3`]: https://github.com/ceramicnetwork/CIPs/blob/main/CIPs/cip-79.md
-[`did:ion`]: https://github.com/decentralized-identity/ion
+[`base58btc`]: https://github.com/multiformats/multibase/blob/master/multibase.csv#L21
 [`did:key`]: https://w3c-ccg.github.io/did-method-key/
-[base32]: https://github.com/multiformats/multibase/blob/master/multibase.csv#L12
+[`secp256k1`]: https://en.bitcoin.it/wiki/Secp256k1
 [browser api crypto key]: https://developer.mozilla.org/en-US/docs/Web/API/CryptoKey
-[canonical collections]: #71-canonical-json-collection
 [capabilities]: https://en.wikipedia.org/wiki/Object-capability_model
 [caps as keys]: http://www.erights.org/elib/capability/duals/myths.html#caps-as-keys
+[certificate capability model]: http://wiki.erights.org/wiki/Capability-based_Active_Invocation_Certificates
```

Comment:
```suggestion
[certificate capability model]: http://wiki.erights.org/wiki/Capability-based_Active_Invocation_Certificates
[CID]: https://docs.ipfs.tech/concepts/content-addressing/
```
Unless the uglier, raw github definition seems more appropriate or canonical for whatever reason? 
https://github.com/multiformats/cid